src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.Calculator.UnitTest/QueryTests.cs

```diff
@@ -113,5 +111,101 @@ public void NoErrorForDivisionByNumberWithDecimalDigits(string typedString)
             Assert.AreEqual(result, "Copy this number to the clipboard");
             Assert.AreEqual(resultWithKeyword, "Copy this number to the clipboard");
         }
+
+        [DataTestMethod]
+        [DataRow("2(1+1)", "2 * (1+1)")]
+        [DataRow("pi(1+1)", "pi * (1+1)")]
+        [DataRow("pilog(100)", "pi * log(100)")]
+        [DataRow("3log(100)", "3 * log(100)")]
+        [DataRow("2e", "2 * e")]
+        [DataRow("(1+1)(3+2)", "(1+1) * (3+2)")]
+        [DataRow("(1+1)cos(pi)", "(1+1) * cos(pi)")]
+        [DataRow("sin(pi)cos(pi)", "sin(pi) * cos(pi)")]
+        [DataRow("2 (1+1)", "2 * (1+1)")]
+        [DataRow("pi  (1+1)", "pi * (1+1)")]
+        [DataRow("pi  log(100)", "pi * log(100)")]
+        [DataRow("3   log(100)", "3 * log(100)")]
+        [DataRow("2 e", "2 * e")]
+        [DataRow("(1+1)  (3+2)", "(1+1) * (3+2)")]
+        [DataRow("(1+1)  cos(pi)", "(1+1) * cos(pi)")]
+        [DataRow("sin  (pi)  cos(pi)", "sin  (pi) * cos(pi)")]
+        [DataRow("2picos(pi)(1+1)", "2 * pi * cos(pi) * (1+1)")]
+        [DataRow("pilog(100)log(1000)", "pi * log(100) * log(1000)")]
+        [DataRow("pipipie", "pi * pi * pi * e")]
+        [DataRow("(1+1)(3+2)(1+1)(1+1)", "(1+1) * (3+2) * (1+1) * (1+1)")]
+        [DataRow("(1+1) (3+2)  (1+1)(1+1)", "(1+1) * (3+2) * (1+1) * (1+1)")]
+        public void RightHumaMultiplicationExpressionTransformation(string typedString, string expectedQuery)
```

Comment:
Huma -> Human. (Same for the other checks.

Reply:
solved